src/models/Schema.ts

```diff
@@ -40,7 +40,7 @@ export class Schema extends CommonSchema<Schema | boolean> {
     contentMediaType?: string; //Enum?
     definitions?: { [key: string]: Schema | boolean; };
     description?: string;
```

Comment:
Did you remove the `required?: string[];` from schema?

Reply:
Nope, I only changed type for `default`.

Reply:
My question goes towards, it should be removed since it is now part of the CommonSchema right?

Reply:
We shouldn't alter JSON Draft7 Schema type. I haven't a problem with `required` field inside schema.

Reply:
But it inherits from CommonSchema which now has the property, same as all the other properties inside CommonSchema are no longer defined inside the Schema class 😄

Reply:
You're right.